backend/src/common/plugins/types/auth-options.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type ServerAppApi } from '~/common/server-application/types/types.js';
+import { type JwtService } from '~/common/services/jwt/jwt.service.js';
+
+type AuthOptions = {
```

Comment:
I would keep AuthPluginOptions probably, since it more descriptive, but this is not critical